editor/src/components/inspector/sections/layout-section/self-layout-subsection/simplified-layout-subsection.tsx

```diff
@@ -29,38 +35,34 @@ export const SimplifiedLayoutSubsection = React.memo(() => {
 
   return (
     <FlexColumn>
-      <InspectorSubsectionHeader style={{ border: 'none' }}>
-        <FlexRow
-          style={{
-            flexGrow: 1,
-            gap: 8,
-            height: 42,
-          }}
-        >
-          <EditorContractDropdown />
-          <ResizeToFitControl />
-        </FlexRow>
-      </InspectorSubsectionHeader>
+      <FlexRow
+        style={{
+          flexGrow: 1,
+          gap: 8,
+          height: UtopiaTheme.layout.rowHeight.normal,
+          padding: `0 ${UtopiaTheme.layout.inspectorXPadding}px 0 4px`,
+          fontWeight: 600,
```

Comment:
I guess this is because `EditorContractDropdown` should be bold (since it's used as a heading)?